cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/mapper/SdkClientExceptionMapper.java

```diff
@@ -16,14 +18,34 @@
     @Inject
     private AwsEncodedAuthorizationFailureMessageDecoder awsEncodedAuthorizationFailureMessageDecoder;
 
-    public RuntimeException map(AwsCredentialView awsCredentialView, String region, SdkClientException e) {
+    public RuntimeException map(AwsCredentialView awsCredentialView, String region, SdkClientException e, Signature signature) {
```

Comment:
This should be unit tested (I'm surprised it does not have any, seems like I forgot to add it in the first place)